email-utils.js

```diff
@@ -76,8 +76,9 @@ const EmailUtils = {
 
     return new Promise((resolve, reject) => {
       gTransporter.use("compile", hbs(hbsOptions));
+      const emailFrom = AppConstants.EMAIL_FROM || `"Firefox Monitor" < ${kSMTPUsername}`;
```

Comment:
This feels suspicious to me now.
Before we had `<${kSMTPUsername}>`, but now we have an extra space after the `<` and it looks like the `>` is missing.

Reply:
Good catch.